### PR TITLE
New keymap

### PR DESCRIPTION
**Pull request by mgalisa**
## Description

Starting a new keymap for the Planck. Tweaked the Raise layer and added an emoji layer to more easily be annoying on Slack.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/planck/keymaps/mgalisa/keymap.c [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+      if (clockwise) {
+        muse_tempo+=1;
+      } else {
+        muse_tempo-=1;
+      }
+    }
+  } else {
+    if (clockwise) {
+      #ifdef MOUSEKEY_ENABLE
+        register_code(KC_MS_WH_DOWN);
```
Comment (fauxpark) [inline keyboards/planck/keymaps/mgalisa/keymap.c]: You can use `tap_code(KC_MS_WH_DOWN)` etc. for these.

Comment (mgalisa) [reply keyboards/planck/keymaps/mgalisa/keymap.c]: Thanks - that piece is from where I copied the default Planck keymap (I haven't actually touched that piece on my own yet.) Is one better than/preferred over the other? If so I could submit a second PR to update the default.

Comment (drashna) [reply keyboards/planck/keymaps/mgalisa/keymap.c]: either works fine, actually.  But, specifically, `tap_code` calls register_code, adds a delay (if defined), and then unregister.  So, it does "mostly" the same thing. 

Also, you don't need to open an additional PR.  Any additional commits on the branch that you opened the PR from (`matt_dev` in this case) will automatically update this PR.

Comment (mgalisa) [reply keyboards/planck/keymaps/mgalisa/keymap.c]: Makes sense - I tested replacing `register_code()`/`unregister_code()` on my own keymap where it makes sense to do so and all seems well. Latest commit includes the changes in `planck/keymaps/default/keymap.c` as well.

**Pull request merged**